angular/src/directives/api-action.directive.ts

```diff
@@ -31,7 +31,7 @@ export class ApiActionDirective implements OnChanges {
         }, (e: any) => {
             this.el.nativeElement.loading = false;
 
-            if (e instanceof ErrorResponse && (e as ErrorResponse).captchaRequired) {
+            if ((e instanceof ErrorResponse || e.constructor.name === 'ErrorResponse') && (e as ErrorResponse).captchaRequired) {
```

Comment:
I was wondering about this, but good catch!

Reply:
I just wish I knew when one would work over the other. Some light googling suggested ES3 or ES5 needing the constructor name, but browser (where the issue popped up) is targeting ES2016